docs/pages/machine-id/guides/jenkins.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Jenkins
+description: Using Teleport Machine ID with Jenkins
+---
+
+Jenkins is a open source automation server that is frequently used to build
+Continuous Integration and Continuous Delivery (CI/CD) pipelines.
+
+In this guide, we will demonstrate how to migrate Jenkins workloads to
+Machine ID with minimal changes to your pipelines.
+
+## Prerequisites
+
+You will need the following tools to use Teleport with Jenkins.
+
+- [Teleport Enterprise](../../enterprise/introduction.mdx) or [Teleport Cloud](../../cloud/introduction.mdx) >= 9.0.0 
+- `ssh` OpenSSH tool
+- `Jenkins`
+
+In addition, if you already have not done so, follow the
+[Machine ID Getting Started Guide](../../docs/pages/machine-id/getting-started.mdx)
+to create a bot user and start Machine ID.
+
+## Architecture
+
+Before we begin, it should be noted that Jenkins is a tool that is notoriously
+difficult to secure. Machine ID is one part of securing your infrastructure,
+but it alone is not sufficient. Below we will provide some basic guidance which
+can help improve the security posture of your Jenkins install.
+
+The simplest Jenkins deployments have the controller and agents run on the
```

Comment:
Do we need to introduce what a controller and agent is?

Reply:
Wouldn't anyone that is coming into this guide already be reasonably familiar with Jenkins?

Reply:
I mean I know what Jenkins is, and I've used lots of CI/CD platforms, but I'm not familiar with controllers and agents. I might land on this page because it's the closest thing we have for "using Machine ID in CI/CD" docs.